science/gds/Portfile

```diff
@@ -1,92 +1,64 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem    1.0
-PortGroup     conflicts_build 1.0
-PortGroup     active_variants 1.1
-PortGroup     python 1.0
 
 name          gds
-version       2.18.7
-revision      5
+version       2.19.1
 categories    science
 platforms     darwin
-maintainers   {ligo.org:ed.maros @emaros} openmaintainer
+maintainers   {@emaros ligo.org:ed.maros} openmaintainer
 license       GPL
 
 description   LSC Algorithm Library
 long_description \
   LIGO Scientific Collaboration Algorithm Library containing core \
   routines for gravitational wave data analysis.
 
-homepage      https://www.lsc-group.phys.uwm.edu/daswg/projects/lalsuite.html
-master_sites  http://software.ligo.org/lscsoft/source/
+homepage      https://git.ligo.org/gds/gds
+master_sites  https://software.igwn.org/lscsoft/source/
 
-patchfiles    patch-std_cplusplus_2017.diff
-
-checksums     rmd160  3b2186057dec5d1e530ee1cefbfd12ab4f976d40 \
-              sha256  e592727d686af1042d775f192531d440600105c12d7cdad63455cfad3fb9913c
+checksums     rmd160  b297ab63927141355fdd3e165b6983c4aac04d1a \
+              sha256  6d53977e7538dcff1cf0f48a56ff7f08c05587b82708f6d3e32989f0b8d1e6dd \
+              size    2425166
```

Comment:
there is a checksum mismatch which causes the build failure - see my [first comment](https://github.com/macports/macports-ports/pull/7498#issuecomment-649494999) in this thread.

Reply:
I had noticed these on my local build after I had done a clean --all. I believe that build 12439 should have them all resolved.